index.html

```diff
@@ -36,7 +36,158 @@ <h2>
             <div>
                <img src="Images/Social Media.png" alt="Social icons">
             </div>
-        
+        </section>
+        <section class="works">
+            <div class="content1">
+                <h1>
+                    My Recent Works
+                </h1>
```

Comment:
- [ ] Kindly I recommend to you use  a header tag `hx` for The title (e.g., "My Recent Works") instead of `h1` tag ,This way will be semantically correct and the screen readers and other technologies will thank you. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.
- NB: There should be just one h1 in an HTML file, for SEO(Search Engine Optimization) and web accessibility purposes.